tyr/servers/mongo/arbiter.py

```diff
@@ -2,8 +2,8 @@
 
 class MongoArbiterNode(MongoReplicaSetMember):
 
-    NAME_TEMPLATE = '{envcl}-rs{replica_set}-{zone}-arb'
-    NAME_SEARCH_PREFIX = '{envcl}-rs{replica_set}-{zone}-'
+    NAME_TEMPLATE = '{envcl}-rs{replica_set}-{location}-arb'
```

Comment:
AWS seems refer to the region + availability zone as placement. do you think that'd be a better term?